src/test/java/integration/TestConfig.java

```diff
@@ -0,0 +1,15 @@
+package integration;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.elasticsearch.repository.config.EnableReactiveElasticsearchRepositories;
+
+
+@Configuration
```

Comment:
@hizmailovich should be `@TestConfiguration`, as far I understand

Reply:
`@TestConfiguration` doesn't work for me, only `@Configuration`